doc/dependencies.md

```diff
@@ -4,40 +4,34 @@ These are the dependencies used by Bitcoin Core.
 You can find installation instructions in the `build-*.md` file for your platform.
 "Runtime" and "Version Used" are both in reference to the release binaries.
 
+## Compiler
+
+Bitcoin Core requires one of the following compilers.
 | Dependency | Minimum required |
 | --- | --- |
 | [Clang](https://clang.llvm.org) | [16.0](https://github.com/bitcoin/bitcoin/pull/30263) |
-| [CMake](https://cmake.org/) | [3.22](https://github.com/bitcoin/bitcoin/pull/30454) |
 | [GCC](https://gcc.gnu.org) | [11.1](https://github.com/bitcoin/bitcoin/pull/29091) |
-| [Python](https://www.python.org) (scripts, tests) | [3.10](https://github.com/bitcoin/bitcoin/pull/30527) |
-| [systemtap](https://sourceware.org/systemtap/) ([tracing](tracing.md))| N/A |
```

Comment:
This doesn't show up as a table when I preview the md file.

Reply:
Needs a blank line after line 9.

Reply:
thanks, I was wondering if there should be a blank line. for me it shows the table correctly (firefox, gh mobile) so I thought when in doubt just keep it compact.

weird that it displays correctly for me but not for you though (also tested with `gh-markdown-preview` locally).